fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -57,14 +66,18 @@ const RollCallOpened = () => {
       EventTags.ROLL_CALL, OpenedLaoStore.get().id.toString(),
       rollCallID, time,
     );
-    requestCloseRollCall(updateId, Array.from(attendees).map((key: string) => new PublicKey(key)))
-      .then(() => {
+
+    // Add the token of the organizer before closing the roll call
+    Wallet.generateToken(lao.id, new Hash(rollCallID)).then((token) => {
+      const attendeesList = Array.from(attendees).map((key: string) => new PublicKey(key));
+      attendeesList.push(token.publicKey);
+      requestCloseRollCall(updateId, attendeesList).then(() => {
```

Comment:
```suggestion
      return requestCloseRollCall(updateId, attendeesList).then(() => {
```

if you chain `Promise`s (without the `async`/`await` syntax), you need to `return` the inner `Promise` for this to work as you expect it. Otherwise the outer promise will consider that the asynchronous work is finished before the inner promise has finished executing, i.e., they wouldn't be chained.

Reply:
Done